src/main/java/fi/metatavu/dcfb/server/rest/ItemsApiImpl.java

```diff
@@ -255,10 +255,23 @@ public Response findItem(UUID itemId) throws Exception {
   }
   
   @Override
-  public Response listItems(String categoryIdsParam, String locationIdsParam, String userIds, String search, Double nearLat,
+  public Response listItems(String categoryIdsParam, String locationIdsParam, String userIdsParam, String search, Double nearLat,
       Double nearLon, Boolean includeExhausted, List<String> sort, Long firstResult, Long maxResults) throws Exception {
-
-    // TODO: userIds
+    
+    List<UUID> userIds = getListParameter(userIdsParam, param -> {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove useless curly braces around statement and then remove useless return keyword ([squid:S1602](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS1602))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=32&issues=AWXSgwp4lgbYnohYHn6D&open=AWXSgwp4lgbYnohYHn6D)

[ISSUE_KEY]: AWXSgwp4lgbYnohYHn6D